grunttasks/l10n-htmllint.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('l10n-htmllint', 'Lint l10n files', [
+    // grunt addition for htmllint
+    'clean',
+    'selectconfig:dist',
+    'l10n-generate-pages',
+    'l10n-json-to-html',
+    'htmllint:l10n'
+    ]
+  );
+
+  grunt.registerMultiTask('l10n-json-to-html', 'Convert l10n JSON files to HTML', function () {
```

Comment:
Can you add a comment about why do we need to this, see example here: https://github.com/mozilla/fxa-content-server/blob/master/grunttasks/l10n-generate-pages.js#L6 we include a few lines of documentation